server/models/Document.ts

```diff
@@ -58,6 +59,7 @@ export const DOCUMENT_VERSION = 2;
 type AdditionalFindOptions = {
   userId?: string;
   includeState?: boolean;
+  rejectOnEmpty?: boolean | Error;
```

Comment:
Had to keep `boolean | Error` instead of just `boolean` in order to please `tsc` :see_no_evil: